SmartDeviceLink-iOS.xcodeproj/project.pbxproj

```diff
@@ -3812,14 +3812,6 @@
 			name = Notifications;
 			sourceTree = "<group>";
 		};
-		5DA3F3611BC448690026F2D0 /* Handler Additions */ = {
```

Comment:
I'm assuming all of this was automatic and you didn't cause these groups to be deleted?